### PR TITLE
ECOM-6377 Add fields to MinimalCourseRunSerializer

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
@edx/ecommerce

### CODE REVIEW

**Comment by schenedx** [comment]
@MatthewPiatetsky Should we be concerned about the performance impact this change makes? Like would the connection between LMS and Discovery go down because of this change? 
I guess, a related question is, does this change increase the database count for related unit tests?

**Comment by MatthewPiatetsky** [comment]
@schenedx yeah I noticed that those tests are failing, specifically the reason is that if we want to retrieve type it is a property. We can't prefetch the property, though we could prefetch seats and potentially use cached_property instead of property. I will do a bit more looking, but not sure exactly what we want to do yet

File: course_discovery/apps/api/serializers.py [hunk 1]
```diff
@@ -335,7 +335,8 @@ def prefetch_queryset(cls):
 
     class Meta:
         model = CourseRun
-        fields = ('key', 'uuid', 'title', 'image', 'short_description', 'marketing_url',)
+        fields = ('key', 'uuid', 'title', 'image', 'short_description', 'marketing_url',
+                  'start', 'end', 'enrollment_start', 'enrollment_end', 'pacing_type', 'type',)
```
Comment (rlucioni) [inline course_discovery/apps/api/serializers.py:342]: You'll want to move the seat prefetching to account for moving type.

``` py
return CourseRun.objects.all().select_related('course').prefetch_related(
    'course__partner',
    Prefetch('seats', queryset=SeatSerializer.prefetch_queryset()),
)
```

**Comment by rlucioni** [comment]
@MatthewPiatetsky after you move the prefetching, can you take a look at the django debug toolbar before and after moving those fields to see what the difference in query counts is?

**Comment by MatthewPiatetsky** [comment]
@rlucioni I queried http://localhost:8008/api/v1/programs/ on master and got 11 queries, with this branch I get 29 (and without the cached_property additions I get 47). Am I missing something here with the prefetching? Or are the properties not taking advantage of it?

**Comment by MatthewPiatetsky** [comment]
@rlucioni Now the query I mentioned above only adds 2 queries. I think the issue was with value_list, using a list comprehension seems to resolve the issue.

**Review by rlucioni — APPROVED** [review]
Very nice @MatthewPiatetsky!

**Review by schenedx — APPROVED** [review]
👍

**Pull request merged**